packages/calling/src/CallingClient/registration/register.ts

```diff
@@ -760,7 +760,7 @@ export class Registration implements IRegistration {
     }
 
     this.clearKeepaliveTimer();
-    this.setStatus(MobiusStatus.DEFAULT);
+    this.setStatus(RegistrationStatus.INACTIVE);
```

Comment:
Same question here as in deleteRegistration

Reply:
Here it needed to be INACTIVE only

Reply:
As discussed, it needs to be IDLE here for a purposeful deregistration by the developer

Reply:
As we discussed yesterday, we cannot change the state to IDLE in deleteRegistration as this gets invoked in restoration and failback cases also